apps/sms/views/conversation/js/startup.js

```diff
@@ -72,18 +68,17 @@
   exports.Startup = {
     init() {
       MessageManager.init();
-      Navigation.init();
       ConversationView.init();
-
-      ConversationView.once('visually-loaded', () => {
-        initLazyDependencies();
-        initHeaders();
-      });
-
       initShims().then(() => {
-        // Temporary workaround, navigation part will be revised in bug 1162030.
-        Navigation.defaultPanel = Navigation.getPanelName() || DEFAULT_PANEL;
-        Navigation.toDefaultPanel(Utils.params(exports.location.hash));
+        // We won't be able to access report and group panels directly because
+        // information.js is not loaded before ConversationView finishes
+        // loading.
+        Navigation.init();
+
+        ConversationView.once('visually-loaded', () => {
```

Comment:
Is there any reason why we don't want to call ```Conversation.once``` outside of this callback before ```initShims```?